Source/Core/Core/Boot/Boot.cpp

```diff
@@ -190,7 +190,7 @@ bool CBoot::Load_BS2(const std::string& _rBootROMFilename)
 		return false;
 
 	// Use zlibs crc32 implementation to compute the hash
-	u32 ipl_hash = crc32(0L, Z_NULL, 0);
```

Comment:
it didn't seem necessary to use the Z_NULL constant here, but wasn't as sure about this one as the 0s and NULLs nearly everywhere else